### ISSUE TITLE
Biber warns about "Invalid field 'day' for entrytype 'online'" when validating datamodel

### ISSUE THREAD

**Issue by ClaudiusL** [plk/biblatex#1053]
This seems to be the same bug as in #571. Biber complains about "Invalid field 'day' for entrytype 'online'" when validating the datamodel.

Consider the following BIB file:

```
%test.bib
@online {Guy2020,
	author 	= {Guy, Test},
	title	= {Some Title},
	date	= {2020-09-20},
	url	= {http://example.org}
}
```
Then validate the datamodel in tool mode:

```
>biber --tool test.bib --validate-datamodel
INFO - This is Biber 2.16 (beta) running in TOOL mode
INFO - Logfile is 'test.bib.blg'
INFO - Looking for bibtex format file 'test.bib'
INFO - LaTeX decoding ...
INFO - Found BibTeX data source 'test.bib'
INFO - Datamodel validation starting
WARN - Datamodel: Entry 'Guy2020' (test.bib): Invalid field 'day' for entrytype 'online'
INFO - Datamodel validation complete
INFO - Overriding locale 'en_US' defaults 'normalization = NFD' with 'normalization = prenormalized'
INFO - Overriding locale 'en_US' defaults 'variable = shifted' with 'variable = non-ignorable'
INFO - Sorting list 'tool/global//global/global' of type 'entry' with template 'tool' and locale 'en_US'
INFO - No sort tailoring available for locale 'en_US'
INFO - Writing 'test_bibertool.bib' with encoding 'UTF-8'
INFO - Output to test_bibertool.bib
INFO - WARNINGS: 1
```

Tested on Windows with biber 2.16 (beta).

**Comment by plk** [comment]
This was only failing in tool mode because the default `biber-tool.conf` data model was out of sync - should be fixed in 2.16 (beta) now.

**Comment by ClaudiusL** [comment]
Jep, I can confirm that this works in 2.16 (beta) now. Thanks for looking into this!
